src/DacpacTool/Program.cs

```diff
@@ -47,6 +47,7 @@ static async Task<int> Main(string[] args)
             {
                 new Option<FileInfo>(new string[] { "--input", "-i" }, "Path to the .dacpac package to deploy"),
                 new Option<string>(new string[] { "--targetServerName", "-tsn" }, "Name of the server to deploy the package to"),
+                new Option<string>(new string[] { "--targetPort", "-tprt" }, "Port number to connect on (leave blank for default)"),
```

Comment:
I think we can have an Option<int> here so that we don't have to do any parsing ourselves.

Reply:
I've pushed a change, but C# isn't my primary language, so let me know if it's not the idiomatic way to do things 🙂